src/EditorFeatures/Core/Implementation/Classification/SyntacticClassificationTaggerProvider.cs

```diff
@@ -15,14 +18,17 @@
 
 namespace Microsoft.CodeAnalysis.Editor.Implementation.Classification
 {
-    [Export(typeof(ITaggerProvider))]
+    [Export(typeof(IViewTaggerProvider))]
     [ContentType(ContentTypeNames.CSharpContentType)]
     [ContentType(ContentTypeNames.VisualBasicContentType)]
     [TextViewRole(PredefinedTextViewRoles.Document)]
     [TagType(typeof(IClassificationTag))]
-    internal partial class SyntacticClassificationTaggerProvider : ITaggerProvider
+    internal partial class SyntacticClassificationTaggerProvider : IViewTaggerProvider
```

Comment:
It does **not** work with RTF-copy or scroll margin as an IViewTaggerProvider. :disappointed: